src/test/FuzzerImpl.cpp

```diff
@@ -1599,6 +1564,42 @@ TransactionFuzzer::initializeOffers(AbstractLedgerTxn& ltxOuter)
     ltx.commit();
 }
 
+void
+TransactionFuzzer::reduceNativeBalancesAfterSetup(AbstractLedgerTxn& ltxOuter)
+{
+    LedgerTxn ltx(ltxOuter);
+
+    xdr::xvector<Operation> ops;
+
+    for (auto const& param : accountParameters)
+    {
+        PublicKey account;
+        FuzzUtils::setShortKey(account, param.mShortKey);
+
+        // Reduce "account"'s native balance by paying the root, so that
+        // fuzzing has a better chance of exercising edge cases.
+        auto ae = stellar::loadAccount(ltx, account);
+        auto const availableBalance = getAvailableBalance(ltx.loadHeader(), ae);
+        auto const targetAvailableBalance =
+            param.mNativeAssetAvailableForTestActivity +
+            FuzzUtils::FUZZING_FEE *
+                FuzzUtils::DEFAULT_NUM_TRANSACTIONS_TO_RESERVE_FEES_FOR;
+
+        if (availableBalance > targetAvailableBalance)
```

Comment:
I thought you were going to handle the "else" cases on these?

Reply:
Good catch; I missed that one!  It's an assert now (we shouldn't be taking an `else` case at all; it would mean that setup wasn't doing what the parameters said it was).